### ISSUE TITLE
Crash with custom colors using Gradle 4 / Android gradle plugin 3

### ISSUE THREAD

**Issue by ANPez** [mikepenz/MaterialDrawer#2094]
I have the theme defined like this:
```
    <style name="AppTheme" parent="MaterialDrawerTheme">
....
        <!-- Drawer -->
        <item name="material_drawer_background">#7d7e7b</item>
        <item name="material_drawer_selected">#4d4d4d</item>
    </style>
```

and it crashes with the error:

```
Caused by: android.content.res.Resources$NotFoundException: Resource ID #0x0
at android.content.res.Resources.getValue(Resources.java:1351)
at android.content.res.Resources.getColor(Resources.java:963)
at android.support.v4.content.res.ResourcesCompat.getColor(ResourcesCompat.java:143)
at com.mikepenz.materialize.util.UIUtils.getThemeColor(UIUtils.java:42)
at com.mikepenz.materialize.util.UIUtils.getThemeColorFromAttrOrRes(UIUtils.java:56)
at com.mikepenz.materialdrawer.DrawerBuilder.buildView(DrawerBuilder.java:1522)
at com.mikepenz.materialdrawer.DrawerBuilder.build(DrawerBuilder.java:1333)
```

If I delete the two lines referring the custom colors, it does not crash, but obviously also does not set the colors. I am using the version 6.0.0 of your library, with Gradle 4.1 and the Android plugin 3.0.0.

**Comment by mikepenz** [comment]
@ANPez fixed in v6.0.1

**Comment by ANPez** [comment]
Great, thanks! Searched for the issue, but didn't find it on the tracker, sorry :3
